Add ability to bind a key to accept line and run it at once
--- Aloxaf---fzf-tab/fzf-tab.zsh	2020-11-04 15:23:44.565964815 -0500
+++ Aloxaf---fzf-tab/fzf-tab.zsh-patch	2020-11-04 15:23:11.025414960 -0500
@@ -95,7 +95,7 @@
 -ftb-complete() {
   local -a _ftb_compcap
   local -Ua _ftb_groups
-  local choice choices _ftb_curcontext continuous_trigger print_query bs=$'\2' nul=$'\0'
+  local choice choices _ftb_curcontext continuous_trigger print_query accept_line bs=$'\2' nul=$'\0'
 
   # must run with user options; don't move `emulate -L zsh` above this line
   (( $+builtins[fzf-tab-compcap-generate] )) && fzf-tab-compcap-generate -i
@@ -116,6 +116,7 @@
       -ftb-generate-header     # sets `_ftb_headers`
       -ftb-zstyle -s continuous-trigger continuous_trigger || continuous_trigger=/
       -ftb-zstyle -s print-query print_query || print_query=alt-enter
+      -ftb-zstyle -s accept-line accept_line || accept_line=enter
 
       # NOTE: Using pipe here causes an error "failed to read /dev/tty"
       # when _ftb_complist is long
@@ -145,6 +146,10 @@
   if [[ $choices[1] && $choices[1] == $continuous_trigger ]]; then
     typeset -gi _ftb_continue=1
   fi
+
+  if [[ $choices[1] && $choices[1] == $accept_line ]]; then
+    typeset -gi _ftb_accept=1
+  fi
   choices[1]=()
 
   for choice in "$choices[@]"; do
@@ -170,6 +175,7 @@
   local -i _ftb_continue=1
   while (( _ftb_continue )); do
     _ftb_continue=0
+    local _ftb_accept=0
     local IN_FZF_TAB=1
     {
       zle .fzf-tab-orig-$_ftb_orig_widget
@@ -181,7 +187,7 @@
       zle .reset-prompt
       zle -R
     else
-      zle redisplay
+      (( _ftb_accept )) && zle accept-line || zle redisplay
     fi
   done
 }
--- Aloxaf---fzf-tab/lib/-ftb-fzf	2020-11-04 15:23:44.565964815 -0500
+++ Aloxaf---fzf-tab/lib/-ftb-fzf-patch	2020-11-04 15:23:38.826034508 -0500
@@ -49,7 +49,7 @@
   --color=hl:$(( header_lines == 0 ? 188 : 255 )) \
   --cycle \
   --delimiter='\x00' \
-  --expect=$continuous_trigger,$print_query \
+  --expect=$continuous_trigger,$print_query,$accept_line \
   --header-lines=$header_lines \
   --height=${FZF_TMUX_HEIGHT:=$(( lines > LINES / 3 * 2 ? LINES / 3 * 2 : lines ))} \
   --layout=reverse \
